app/views/wpcc/your_results/_period_percents_table.html.erb

```diff
@@ -11,21 +11,21 @@
       <thead>
         <tr>
           <th><span class="visually-hidden"><%= t('wpcc.results.period_title.contributor_title') %></span></th>
-          <% period_legal_percents.each do |period| %>
+          <% period_legal_percents.tap do |period| %>
```

Comment:
Is it really necessary to use tap, it kind of adds an element of abstraction?

Reply:
We don't seem to need/be returning the object after operating over its attributes/contents, that is what `tap` helps with.